tools/vsimporter/xib2nib/NIBWriter.cpp

```diff
@@ -302,21 +302,47 @@ XIBObject *NIBWriter::GetProxyFor(XIBObject *obj)
 }
 
 std::map<std::string, std::string> _g_exportedControllers;
+std::map<std::string, std::string> _g_exportedControllersbyStoryboardId;
```

Comment:
Not sure what you mean by the reference platform entries. I thought the info.plist file is a concept introduced by you to manage the nibs that are generated from a storyboard.